tensorflow_addons/metrics/README.md

```diff
@@ -0,0 +1,34 @@
+# Addons - Metrics
+
+## Maintainers
+| Submodule  | Maintainers  | Contact Info   |
+|:---------- |:------------- |:--------------|
+|  |  |  |
+
+## Contents
+| Submodule | Activation  | Reference                               |
+|:----------------------- |:-------------------|:---------------|
+|  |  |            |
+
+
+## Contribution Guidelines
+#### Standard API
+In order to conform with the current API standard, all metrics
+must:
+ * Be a `tf.function`.
+ * Have the signature `fn(input, axis=-1, name=None)`.
```

Comment:
Metric must be subclass of `tf.metrics.Metric`, as Sean pointed. I think we should replace the two lines by: Inherit from `tf.metrics.Metric`.